### DISCUSSION TITLE
Bug: Stargazers for a repository is not showing

### DISCUSSION THREAD

**Discussion by anantajitjg** [community/community#8641] [Other Feature Feedback, Questions, & Ideas]
Before introducing the Lists feature, I could view the stargazers for a repository. But now, this feature is not available when l login. For the public view, this feature is available. So I believe this is a possible bug.

**Public View**

![Public](https://user-images.githubusercontent.com/22009263/145525349-cb66f68d-b78f-484a-88ab-03fbb0641f11.png)

**After Login**

![Login](https://user-images.githubusercontent.com/22009263/145525413-4991cc45-ec41-4fe3-8b96-7943557e97f2.png)

**Comment by cszach** [comment]
Hello.

You can still see the stargazers of a repository by clicking on an entry listed under its _About_ section.

![image](https://user-images.githubusercontent.com/24489228/145528060-5621110e-0207-4227-9748-be25e18fe3dd.png)

**Reply by anantajitjg** [reply]
Thanks for this info. But earlier, it was easy and very noticeable. I hope it will be brought back  🤞
